renovate.json5

```
@@ -0,0 +1,25 @@
+{
+  // start with default Renovate settings
+  "extends": [
+    "config:base"
+  ],
+
+  // rules for specific subsets of packages
+  "packageRules": [
+    // for everything except major updates
+    {
+      "updateTypes": ["minor", "patch", "pin", "digest", "lockFileMaintenance"],
+
+      // automatically merge, as long as tests pass
+      "automerge": true
```

```suggestion
      "automerge": false
```

let's review at least 3-4 of those merge request manually to be sure - before we automate it fully 🤗